circle.yml

```diff
@@ -1,3 +1,10 @@
 machine:
   node:
-    version: 5.0.0
\ No newline at end of file
+    version: 5.0.0
+deployment:
+  npm:
+    branch: master
+    commands:
+      # login using environment variables
+      - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login
+      - npm run 2npm
```

Comment:
@rodrigopavezi Do we need a module to do this? We should be able to publish with just `npm publish`.